Content.Server/Tiles/FloorTileSystem.cs

```diff
@@ -49,10 +50,25 @@ private void OnAfterInteract(EntityUid uid, FloorTileComponent component, AfterI
                 if (physics.TryGetComponent(ent, out var phys) &&
                     phys.BodyType == BodyType.Static &&
                     phys.Hard &&
-                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0)
-                    return;
+                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0) 
+                    {
+                        // e.g. you can place tile under directional windows
+                        var gridUid = location.GetGridUid(EntityManager);
+                        var floorPos = location.ToMapPos(EntityManager, _transform);
+                        var floorBox = Box2.UnitCentered.Translated(floorPos);
+                        if (TryComp<MapGridComponent>(gridUid, out var comp))
+                        {
+                            floorBox.Scale(comp.TileSize);
+                        }
+                        if (!TryComp<FixturesComponent>(ent, out var fixtures) || !TryComp<TransformComponent>(ent, out var transform))
+                            return;
+                        var wallPos = transform.Coordinates.ToMapPos(EntityManager, _transform);
```

Comment:
Don't convert to mapcoordinates and use AABBs in any circumstance because you will get lots of precision issues with rotated grids, get it in grid terms.